src/components/nodes/taskNode.js

```diff
@@ -0,0 +1,19 @@
+import Node from './node';
+import cloneDeep from 'lodash/cloneDeep';
+
+export default class TaskNode extends Node {
+  clone(nodeRegistry, moddle, $t) {
+    let definition = nodeRegistry[this.type].definition(moddle, $t);
+    const diagramClone = cloneDeep(this.diagram);
```

Comment:
Is cloning the diagram like this required? I had issues with lodash's `clone` and `cloneDeep` methods as it didn't copy everything correctly. Do you need to copy more than what's specified in `Node.diagramPropertiesToCopy`?

Reply:
With this cloning, it seems to get additional configurations on tasks. Like `DueIn`, `Users` & `Assignment Rules`. And for the `connector-send-email`, it copies the `Email Notifications`.

Reply:
Lodash's clone methods seem to modify the object in unexpected ways:

<img width="808" alt="Screen Shot 2020-03-13 at 11 26 26 AM" src="https://user-images.githubusercontent.com/7561061/76635220-7d5e2400-651d-11ea-8600-97dd76c6bb9b.png">

You can see above that the regular element is referred to as `ModdleElement` and has some additional properties, but when cloned, it's now simply `Base` and is missing a few things from before. This may be related to the `isGeneric` error I experienced.